### ISSUE TITLE
Update documentation to show how to access userdb

### ISSUE THREAD

**Issue by andyhasit** [colinskow/superlogin#124]
It says:

> You can also use the same token and password combination to access your personal database.

But doesn't tell you how to do that. I've tried:

 - /auth/dbname$username
 - /dbname$username
 - The long url under userDBs returned in the session.

All tried from postman with `Authorization": "Bearer {token}:{password}"` to no avail. Perhaps I'm doing something else wrong, but if the docs could at least confirm what the correct url should look like (with an example, perhaps) that would be great.

**Comment by colinskow** [comment]
The long URL returned when you open a session should be the correct URL to access the DB. If that's not working, please provide the error and info on your CouchDB setup.

**Comment by andyhasit** [comment]
Thanks for jumping in! A slight problem with that url is that points to the couchdb port (localhost:5984) which is not going to be open...

**Comment by colinskow** [comment]
SuperLogin is designed to give the user direct access to a publicly
available CouchDB instance. Otherwise you can use the bearer token to
access an Express endpoint.

**Comment by andyhasit** [comment]
Ok.... well that explains a lot! I had it in my head that Express+Superlogin would act as the main server and route calls to a private CouchDB endpoint hidden behind it, intercepting them to check authorisation. I assumed this because it's what you'd normally do: hide your db behind your web server and NEVER allow public access to it.
On reflection this all makes sense now, I'd forgotten what CouchDB is. Bear in mind I only came to Superlogin because it allegedly makes auth a breeze for CouchDB, and I only came to CouchDB because I want to replicate private user data from PouchDB clients. There have to be 1000's of people who come to Superlogin from that exact same angle, and it wouldn't surprise me if many of them made the same mistake I did. 
It really wouldn't take much for the readme to clarify this, preferably right at the top! And also mention that if admin part is enabled, it will all look like its working even when it isn't because you have full access anyway (nice gotcha there).

After disabling admin party, and setting the admin credentials in Superlogin config, I'm now getting {"error":"unauthorized","reason":"You are not authorized to access this db."} when I try to access a user database, with what I believe to be the correct Bearer token, so I'm a bit stumped.

Also: the URL includes the token and password, surely this isn't wise, or even necessary, considering we also pass those in the Authorization header?

**Comment by micimize** [comment]
@andyhasit It'll be easier to diagnose your authentication problem with the actual code/config you're using
 
If you're using pouch + ssl, I don't believe including user/pass in the url is a security risk. I don't believe pouch sends the user/pass on every request. Rather, `new PouchDB(session.userDBs.private)` probably authenticates the session, then uses the resultant token. You can also use [pouch-authentication](https://www.npmjs.com/package/pouchdb-authentication), though I can't think of why that'd be necessary when using superlogin.

**Comment by andyhasit** [comment]
@micimize you are correct, the user/pass in the URL gets stripped from the URL that gets sent over the wire, and is encrypted instead, which I did not know at the time. 
The source of my error, was that I was including the user/pass as headers in the couchdb call, similar to how you obtain the token in the first place, and these were clashing with the user/pass in the URL. I'll send a PR once I get back onto this project and have my superlogin up and running just to make sure I have everything right in my mind as to how it works.
